web-performance/README.md

```diff
@@ -0,0 +1,28 @@
+Web Performance
+===============
+
+Web performance refers to the speed in which web pages are downloaded and
+displayed on the user's web browser. Web performance optimization (WPO) or
+website optimization is the field of knowledge about increasing web performance.
+
+
+Resources
+---------
+
+* [Demystifying Speed Tooling (Google I/O ’19)](https://www.youtube.com/watch?v=mLjxXPHuIJo)
+* [Fast load times](https://web.dev/fast)
+* [Your first performance budget with Lighthouse](https://bitsofco.de/your-first-performance-budget-with-lighthouse/)
+* [Visualise performance impact between deploys (with Calibre)](https://calibreapp.com/blog/visualise-performance-impact-between-deploys/)
+* [CSS Wizardy Archive](https://csswizardry.com/archive/) - A reverse-
+  chronological list of over 200 posts from CSS Wizardry covering CSS,
+  architecture, performance, and web development.
+
+
+Tools
+-----
+
+* [Google Lighthouse](https://developers.google.com/web/tools/lighthouse/) -
+  Lighthouse is an open-source, automated tool for improving the quality of web
+  pages.
+* [Calibre](https://calibreapp.com/) - World-class web performance monitoring
+  for teams; powered by Google Lighthouse.
```

Comment:
“World-class” feels like marketing jargon for Calibre - thoughts on reframing this as to its value to us?

```
* [Calibre](https://calibreapp.com/) - Automated performance monitoring, with
  device emulation, metric budgets, and alerts (powered by Lighthouse)
```

Reply:
Totally agree. But I have to say, your opinion here is world-class.